libraries/AC_Planck/AC_Planck.cpp

```diff
@@ -180,10 +182,12 @@ void AC_Planck::request_land(const float descent_rate)
 }
 
 //Move the current tracking target, either to an absolute offset or by a rate
-void AC_Planck::request_move_target(const Vector3f offset_cmd_NED, const bool is_rate)
+void AC_Planck::request_move_target(const Vector3f offset_cmd_NED, const bool is_rate, const float rate_up, const float rate_down)
 {
   //all directions and are valid
   uint8_t valid = 0b00000111;
+  uint32_t max_rates = uint8_t(rate_up) << 8;
```

Comment:
Maybe make this packing a method since its done in two separate locations?

Reply:
changed